app/views/products/_product_form.html.erb

```diff
@@ -0,0 +1,109 @@
+      <%= f.text_field :name, :class => "form-control", :placeholder => "Name", :required => true %>
+    </div>
+    <div class = "col-sm-2" >* required</div>
+  </div>
+
+  <div class = "form-group">
+    <%= f.label :version, "Version", :class => "control-label col-sm-2" %>
+    <div class = "col-sm-4">
+      <%= f.text_field :version, :class => "form-control", :placeholder => "Version"%>
+    </div>
+  </div>
+
+  <div class = "form-group">
+    <%= f.label :description, "Description", :class => "control-label col-sm-2" %>
+    <div class = "col-sm-4">
+      <%= f.text_area :description, :class => "form-control", :placeholder => "Write product description here..."%>
+    </div>
+  </div>
+
+  <div class = "form-group">
+    <%= f.label :ehr_type, "EHR Type", :class => "control-label col-sm-2" %>
+    <div class = "col-sm-2">
+      <%= f.radio_button :ehr_type, "Hospital", disabled: f.object.persisted?, :class => "hide" %>
+      <%= f.label :ehr_type_hospital, "Eligible Hospital", class: (f.object.persisted? ? "cannot_select btn btn-default" : "btn btn-default") %>
+    </div>
+    <div class = "col-sm-2">
+      <%= f.radio_button :ehr_type, "Provider", disabled: f.object.persisted?, :class => "hide" %>
+      <%= f.label :ehr_type_provider, "Eligible Provider", class: (f.object.persisted? ? "cannot_select btn btn-default" : "btn btn-default") %>
+    </div>
+    <div class = "col-sm-2" >* required</div>
+  </div>
+
+  <div class = "row">
+    <label class = "control-label col-sm-2">Certification Types</label>
+    <div class = "col-sm-4">
+      <%= f.check_box :c1_test, disabled: f.object.persisted?, :class => "hide" %>
+      <%= f.label :c1_test, "C1 Test - Record and Export", class: (f.object.persisted? ? "cannot_select btn btn-default" : "btn btn-default") %>
+    </div>
+    <div class = "col-sm-2" >* required</div>
+  </div>
+  <div class = "row">
+    <div class = "col-sm-2"></div>
+    <div class = "col-sm-4">
+      <%= f.check_box :c2_test, disabled: f.object.persisted?, :class => "hide" %>
+      <%= f.label :c2_test, "C2 Test - Import and Calculate", class: (f.object.persisted? ? "cannot_select btn btn-default" : "btn btn-default") %>
+    </div>
+  </div>
+  <div class = "row">
+    <div class = "col-sm-2"></div>
+    <div class = "col-sm-4">
+      <%= f.check_box :c3_test, disabled: f.object.persisted?, :class => "hide" %>
+      <%= f.label :c3_test, "C3 Test - Data Submission", class: (f.object.persisted? ? "cannot_select btn btn-default" : "btn btn-default") %>
+    </div>
+  </div>
+  <div class = "row">
+    <div class = "col-sm-2"></div>
+    <div class = "col-sm-4">
+      <%= f.check_box :c4_test, disabled: f.object.persisted?, :class => "hide" %>
+      <%= f.label :c4_test, "C4 Test - Data Filtering", class: (f.object.persisted? ? "cannot_select btn btn-default" : "btn btn-default") %>
+    </div>
+  </div>
+
+  <div class = "form-group">
+    <div class = "col-sm-2"></div>
+    <div class = "col-sm-1">
```

Comment:
If you leave the EHR type blank when creating a new product you get this message "Ehr type can't be blank."  After this message is displayed, you cannot highlight an EHR type.  Your allowed to select the EHR type (and it will save correctly), however it does not show up with the purple highlight.